website/docs/tutorial/00-overview.md

```diff
@@ -0,0 +1,20 @@
+---
+sidebar_label: Overview
+---
+# Tutorial Overview
+
+This tutorial will help you learn how to use [Scaffold Stellar](https://github.com/theahaco/scaffold-stellar) to build and manage smart contracts on the Stellar blockchain and a decentralized application (dApp) to interact with them. Scaffold Stellar is a developer toolkit that provides CLI tools, contract templates, and a starter React UI to get your idea out of your head and on to the network as fast as possible.
+
+:::tip
+If you just want to get up and running quickly, check out the [Quick Start](/docs/quick-start.mdx) guide.
+:::
+
+Before jumping in, you should have a basic understanding of the command line and of general programming concepts, but we'll walk through all the code together so don't worry if you're new to Stellar, Rust, or dApp development. We'll link out to [The Rust Programming Language book](https://doc.rust-lang.org/stable/book/) to explain concepts if you want to dive deeper.
```

Comment:
We could also link to the official docs here and recommend people to take some time there https://developers.stellar.org